tests/test_piiremover.py

```diff
@@ -79,30 +79,35 @@ def test_pii_remover_catches_GB_passports(self):
             self.assertTrue(pii_remover(feedback) == success)
 
 
-    def test_pii_remover_catches_dates(self): 
+    def test_pii_remover_catches_date(self): 
         
         '''
         '''
 
         test_cases = ['21/10/1964','21-12-2067','12 01 1876','12 Feb 2061']
         
-        success = 'This: %s is a specimen date.' % '[PII Removed]'
+        success = 'This: %s is a specimen date.' % '{{ DATE }}'
         
         for i in test_cases:
 
             feedback = 'This: %s is a specimen date.' % i
  
             self.assertTrue(pii_remover(feedback) == success)
 
-    def test_pii_remover_redacts_digits(self): 
+    def test_pii_remover_catches_emails(self): 
         
         '''
         '''
 
-        feedback = 'This is a test (%s) with some redacted numbers (%s), and %s.' % ('1', '23', '456')
-        success = 'This is a test (%s) with some redacted numbers (%s), and %s.' % ('X', 'XX', 'XXX')
+        test_cases = ['a.n.other@user.com','user@example.com']
         
-        self.assertTrue(pii_remover(feedback) == success)
+        success = 'This: %s is a specimen email.' % '{{ EMAIL }}'
+        
+        for i in test_cases:
```

Comment:
Loops in tests are not recommended - best to have separate test cases for each case.

Tests are the opposite of main cade in this respect - being explicit is better than DRY. Copy and pasting is good in tests. This makes each test self contained to run and read, which all helps.